cpp/build-support/run-test.sh

```diff
@@ -60,6 +60,9 @@ rm -f $LOGFILE $LOGFILE.gz
 
 pipe_cmd=cat
 
+# Install the gcs testbench
+sh ../../../../../ci/scripts/install_gcs_testbench.sh default
```

Comment:
I think this is quite confusing without following the previous code in detail. I think doing this in relation to `$ROOT` would make it a bit clearer?
```suggestion
sh $ROOT/../ci/scripts/install_gcs_testbench.sh default
```